cli/args/flags.rs

```diff
@@ -1357,6 +1370,30 @@ Show documentation for runtime built-ins:
             .help("Output documentation in JSON format")
             .action(ArgAction::SetTrue),
         )
+        .arg(
+          Arg::new("html")
+            .long("html")
+            .help("UNSTABLE: Output documentation in HTML format")
```

Comment:
Not sure if it's worth putting `UNSTABLE:` here

Reply:
no, dont think so